tests/block_header_tests.move

```diff
@@ -15,18 +16,23 @@ fun block_header_test() {
     let header = new_block_header(raw_header);
 
     // verify data extract from header
-    assert!(header.version() == to_u32(x"00008020"));
-    assert!(header.parent() == x"0e102b98a160f4416c8ff0198db9b177523525c9de8a00000000000000000000");
-    assert!(
-        header.merkle_root() == x"3b9b941003024e1afa90199732fdb1366a122ab0a5cacd3f7bcb8cb8815a811b",
+    assert_eq!(header.version(), to_u32(x"00008020"));
```

Comment:
maybe lets add info where this data come from

Reply:
We already have this. This is at the beginning of the test. Github just doesn't show this here.